app/models/publications/monthly_statistics/describe_queries.rb

```diff
@@ -0,0 +1,40 @@
+module Publications
+  module MonthlyStatistics
+    module DescribeQueries
+      delegate :candidate_headline_statistics_query,
+               :age_group_query,
+               :sex_query,
+               :area_query,
+               :phase_query,
+               :route_into_teaching_query,
+               :primary_subject_query,
+               :secondary_subject_query,
+               :provider_region_query,
+               :provider_region_and_subject_query,
+               :candidate_area_and_subject_query,
+               to: ::DfE::Bigquery::ApplicationMetrics
+
+      def describe
+        {
+          candidate_headline_statistics_query: candidate_headline_statistics_query(cycle_week:),
+          age_group_query: age_group_query(cycle_week:),
+          sex_query: sex_query(cycle_week:),
+          area_query: area_query(cycle_week:),
+          phase_query: phase_query(cycle_week:),
+          route_into_teaching_query: route_into_teaching_query(cycle_week:),
+          primary_subject_query: primary_subject_query(cycle_week:),
+          secondary_subject_query: secondary_subject_query(cycle_week:),
+          provider_region_query: provider_region_query(cycle_week:),
+          provider_region_and_subject_query: provider_region_and_subject_query(cycle_week:),
+          candidate_area_and_subject_query: candidate_area_and_subject_query(cycle_week:),
+        }.each do |key, value|
+          # rubocop:disable Rails/Output
```

Comment:
I think I mentioned this in an older PR, but why not `Rails.logger.info` so you don't have to disable rubocop?

Reply:
I don't mind either but using rails logger I will have to enter on the logs to see so not sure the real benefit of the logger over the rubocop lines. This code was supposed to be more a helper so I don't go crazy seeing the bigqueries queries list that we have. Usually I am using on the rails console and I am even thinking to create a rails task if necessary.

Reply:
I had a feeling that `RAILS_LOG_TO_STDOUT` would do the trick, but I think because we are using the semantic logger, it will not work. Just leave it as it is I guess.

Reply:
Done